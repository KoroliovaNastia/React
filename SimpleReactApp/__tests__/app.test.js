import React from 'react';
import { shallow, configure } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import User from '../src/user';

configure({ adapter: new Adapter() });

describe('User', () => {
    it('should render correctly', () => {
        const output = shallow(
            <User firstName="Nastia" lastName="Koroliova" />
        );
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});

// describe('Root', () => {
//     it('should render ')
// })