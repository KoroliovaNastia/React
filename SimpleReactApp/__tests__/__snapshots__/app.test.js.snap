// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`User should render correctly 1`] = `
<span>
  User: 
  Nastia
   
  Koroliova
</span>
`;
